recipes/libsecret/all/conanfile.py

```diff
@@ -34,6 +47,11 @@ def validate(self):
     def config_options(self):
         if self.settings.os == "Windows":
             del self.options.fPIC
+        if self.options.with_libgcrypt and self.settings.os != "Linux":
```

Comment:
Need to move it to `configure()` method. When `config_options` is called, the `self.options.with_libgcrypt` is not processed yet, so its value will be the default option value always. On the other hand, `configure()` is one step later and all options are processed there.

Reply:
Oh! I misunderstood that. Thanks for the explanation.